libs/ui/src/lib/components/button/button.tsx

```diff
@@ -23,7 +23,8 @@ export function Button({ children, button, type, disabled, onClick }: ButtonProp
           </span>
         </>
       )}
-      {button === 'secondary' && children}
+
+      {button !== 'primary' && children}
```

Comment:
Maybe you can revert it, it's easier to understand when is secondary button instead not primary button

Reply:
Changes were done, if you can review it, thanks for the review man